src/reducers/scenes/SettingsReducer.ts

```diff
@@ -94,6 +99,7 @@ export const settingsLegacy = (state: SettingsState = initialState, action: Acti
         mostRecentWallets,
         passwordRecoveryRemindersShown,
         userPausedWallets,
+        userPausedWalletSet: new Set(userPausedWallets),
```

Comment:
userPausedWalletsSet to make it more obvious it's the same as `userPausedWallets`

Reply:
Done